djangoProject/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.4-slim
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the requirements file into the container
+COPY requirements.txt /app/
+
+# Install dependencies
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy the rest of the project files into the container
+COPY . /app/
```

Comment:
Avoid copying sensitive files (e.g., `.env`, `.git`, or cache files).
Use a `.dockerignore` file to exclude files that aren’t required in production.